tests/Instances.hs

```diff
@@ -27,8 +28,15 @@ instance Arbitrary Text where
 instance (Ord k, Arbitrary k, Arbitrary v) => Arbitrary (Map.Map k v) where
     arbitrary = Map.fromList <$> arbitrary
 
+instance Arbitrary TimeOfDay where
+    arbitrary = do
+      Positive h <- arbitrary
+      Positive m <- arbitrary
+      Positive s <- arbitrary
+      return $ fromMaybe (TimeOfDay 0 0 0 ) (makeTimeOfDayValid h m s)
+
```

Comment:
Hi @benweitzman , thanks for the PR and sorry for the long delay in reviewing this.

I think the following Arbitrary instance for TimeOfDay is a bit better since it doesn't generate the default `TimeOfDay 0 0 0` when the generated TimeOfDay is invalid:
  
```haskell
instance Arbitrary TimeOfDay where
  arbitrary = do
    h <- choose (0,23)
    m <- choose (0,59)
    s <- choose (0,59)
    return $ TimeOfDay h m s
```

If you agree, could you change that and update your PR so I can merge it.